assets/js/show-task.js

```diff
@@ -23,6 +24,10 @@ function showTask(inputValue, formattedDate, taskList, form, allTasks, id) {
   const date = document.createElement("p");
   date.innerHTML = "Created At: " + formattedDate;
 
+  const saveButton = document.createElement("button");
+  saveButton.innerText = "Save";
```

Comment:
an utility wrapper function over `document.createElement` can be created and reused in different places.

```suggestion
  const date = createElement("p", "Created At: " + formattedDate);

  const saveButton = createElement("button", "Save");
```

```
function createElement(tag, text) {
    const $e = document.createElement(tag)
    $e.innerText = text

    return $e
}
```